content/functions/cond.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "cond"
+date: 2017-09-08
+description: "Return one of two arguments, depending on the value of a third argument."
+categories: [functions]
+menu:
+  docs:
+    parent: "functions"
+signature: ["cond CONTROL VAR1 VAR2"]
+aliases: [/functions/cond/]
+hugoversion: 0.27
+relatedfuncs: [default]
+toc: false
+draft: false
+needsexamples: false
+---
+
+`cond` returns *VAR1* if *CONTROL* is true, or *VAR2* if it is not.
+
+Example:
+
+```
+{{ cond (eq (len $geese) 1) "goose" "geese" }}
```

Comment:
`cond (len $geese | eq 1) ...` might be a bit easier to read

Reply:
I disagree. I find the pipe syntax to be quite overused. It's great for thinking about stuff functionally, but for stuff like this it's often easier not to think about it functionally.

Reply:
I agree with @jetwash in this case about the pipe (which is very useful in other cases), but the prime example, in this case, is to conditionally set a variable based on some condition.